theme/layouts/_breadcrumbs.ejs

```diff
@@ -0,0 +1,23 @@
+<%
+const query = site.queryPages;
+if (page.pathname !== '') {
+  const queryPage = query(({pathname}) => page.pathname === pathname);
+  const names = queryPage[0].originalPathname.split('/'); %>
+  
+  <nav aria-label="breadcrumb" id = 'breadcrumb'>
+  <ol>
+    <% for (const name of names) {
+      queryEachName = query(({originalPathname}) => 
+        name == originalPathname.split('/')[originalPathname.split('/').length-1]);
+      lastBread = name == names[names.length -1];
+      if (lastBread) { %>
+        <span aria-current='page'><%= queryEachName[0].title %></span>
+      <% }  else { %>
+        <li>
+        <a href = '<%= queryEachName[0].pathname %>'><%= queryEachName[0].title %></a>
+        </li>
+    <% } 
+    } %>
+  </ol>
+</nav>
```

Comment:
Deatail: In current file you are enclosing strings in both `"` and `'` while in other files it's `"`. Let's keep consistent those.